source/topics/paas/how-to-download-database-backup.md

```diff
@@ -0,0 +1,6 @@
+To download a database backup:
+
+1. [Restore the backup to a new database](paas/how-to-restore-from-database-backup).
```

Comment:
That need link to starts with a `/` or it'll be relative to this page (and not work)